frontend/src/components/BackButton.tsx

```diff
@@ -0,0 +1,25 @@
+import BackIcon from "../assets/backIcon.png";
+
+const BackButton = () => {
+  return (
+    <button
+      style={{
+        width: "45px",
+        height: "45px",
+        display: "inline-flex",
+        alignItems: "center",
+        justifyContent: "center",
+      }}
+      className={"rounded-full bg-secondary-bg-500"}
```

Comment:
no need for escape braces here since just a string
```suggestion
      className="rounded-full bg-secondary-bg-500"
```